### ISSUE TITLE
istio-sidecar.deb (1.3.3) is not a debian format archive

### ISSUE THREAD

**Issue by Docteur-RS** [istio/istio#17882]
**Bug description**
Installing istio-sidecar.deb (1.3.3) is not working with the following error message : 
```
sudo dpkg -i istio-sidecar.deb
dpkg-deb: error: `istio-sidecar.deb' is not a debian format archive
dpkg: error processing archive istio-sidecar.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 istio-sidecar.deb
```
**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[ ] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**
Package shoud install correctly.  
Operation works with the previous version (1.3.2)

**Steps to reproduce the bug**
```shell
$ curl -L https://storage.googleapis.com/istio-release/releases/1.3.3/deb/istio-sidecar.deb > istio-sidecar.deb

$ sudo dpkg -i istio-sidecar.deb
dpkg-deb: error: `istio-sidecar.deb' is not a debian format archive
dpkg: error processing archive istio-sidecar.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 istio-sidecar.deb
```

**Version (include the output of `istioctl version --remote` and `kubectl version`)**
As this is for mesh expansion its out of the scope of any cluster

**Environment where bug was observed (cloud vendor, OS, etc)**
Debian, on premise

**Comment by howardjohn** [comment]
I think this is fixed in 1.4, https://gcsweb.istio.io/gcs/istio-build/dev/1.4-alpha.0039742337ddf3b766ac974e9a7aad003896cfcf/deb/ looks to be the right format.

**Comment by Docteur-RS** [comment]
@howardjohn Thanks !
